R/harmony_option.R

```diff
@@ -0,0 +1,98 @@
+#' Set advanced options for HarmonyMatrix
+#' @param lambda Default lambda = c(0.1, 10). In the new release of Harmony,
```

Comment:
The docstring of lambda is not clear.

1. Describe what lambda corresponds to.
2. Mention that this parameter accepts a range and the parameter is being set dynamically using heuristics during runtime.
3. Mention that a trivial range [1,1] will disable the heuristics and lambda will be fixed.

Reply:
OK